go-controller/pkg/testing/mock_ovn.go

```diff
@@ -64,10 +68,11 @@ var _ goovn.Execution = &MockExecution{}
 // return a new mock client to operate on db
 func NewMockOVNClient(db string) *MockOVNClient {
 	mock := &MockOVNClient{
-		db:         db,
-		cache:      make(map[string]MockObjectCacheByName),
-		errorCache: make(map[string]error),
-		connected:  true,
+		db:              db,
+		cache:           make(map[string]MockObjectCacheByName),
+		errorCacheMutex: sync.Mutex{},
```

Comment:
`sync.Mutex`es intentionally do not need to be initialized

Reply:
I didn't know that. It means we could probably clean that up in some other places around our code which initializes the mutexes

Reply:
I think that Dan means that you don'need to initialize  `errorCacheMutex` in the constructor, just remove this line, right Dan?
xref https://golang.org/doc/effective_go.html#allocation_new

Reply:
ups, sorry, my bad, I didn't understand your comment :sweat_smile: